classes/image_types_tegra.bbclass

```diff
@@ -540,7 +540,7 @@ do_image_tegraflash[depends] += "zip-native:do_populate_sysroot dtc-native:do_po
                                  virtual/kernel:do_deploy ${TEGRAFLASHDEPS} \
                                  ${@'${INITRD_IMAGE}:do_image_complete' if d.getVar('INITRD_IMAGE') != '' else  ''} \
                                  ${@'${IMAGE_UBOOT}:do_deploy ${IMAGE_UBOOT}:do_populate_lic' if d.getVar('IMAGE_UBOOT') != '' else  ''} \
-                                 ${@'cboot:do_deploy' if d.getVar('IMAGE_UBOOT') != '' and d.getVar('SOC_FAMILY') == 'tegra186' else  ''}"
+                                 cboot:do_deploy"
```

Comment:
This change looks like it may break TX1 builds.  Changing the `and d.getVar...` conditional to be `!= 'tegra210'`, instead of removing it altogether, might be the right way to fix it.

Reply:
Good point! I changed it